_taxonomy.rst

```diff
@@ -0,0 +1,69 @@
+.. _taxonomy:
+
+Documentation Taxonomy
+======================
+
+Data Management uses and produces multiple classes of documentation.
+Each class fulfills a specific role, with a consistent management and technical process.
+
+.. _taxonomy-outline:
+
+Document classes
+----------------
+
+The classes of DM documentation are:
+
+Requirements and interface control documentation (LPM, LSE)
+   These documents specify functionality that the Data Management System must deliver to the Project.
+   Similarly, **interface control documents** are agreements between subsystems on functionality that cross subsystem boundaries.
+   The production of this documentation is described in `LPM-19: Change Control Process`_ and its management is described in `LPM-51: Document Management Plan`_.
+
+Change-controlled DM design documentation (LDM)
+   These documents define the **scope** and **budget** of Data Management work that meet the **requirements** and **interface control documentation**.
+   DM work is expected to be consistent with these **design documents**.
+   LDM document authors are responsible for updating design documentation whenever necessitated by revised project requirements, or by evolving implementation choices.
+   Design documents must be approved by the Change Control Board (`LPM-19: Change Control Process`_) before becoming the new baseline.
+   These documents are also governed by `LPM-51: Document Management Plan`_.
+   See :ref:`ldm` for how change-controlled documents are handled by the Data Management Documentation Architecture.
+
+Technical notes (DMTN, SQR)
+   Technical notes are a standardized, durable, document series allowing useful information to be published on demand by any Data Management team member wishing to capture and share knowledge.
+   Common uses of **technical notes** are descriptions of experiments with code or data, reviews of literature or technologies, usage notes, or technical documents that are too limited to trigger the change-control process.
+   The primary audiences of **technical notes** are DM team members who build, maintain, or interface with a system and the science community.
+   See :ref:`technotes` for details.
+
+User Guides (DMG)
+   These documentation products describe usage of DM software, platforms, and data products to end-users.
+   Typically, end-users are astronomers in the scientific community.
+   Some **user guides** may instead be considered internal, such as **operational guides** or **developer guides**.
```

Comment:
In what way do you see operational manuals as distinct from user guides?

The basic axes in the documentation taxonomy are:

1. **Authoring policy/controls.** User guides are written by the DM team as part of work tickets. I imagine the same is true for operational manuals.
2. **Format.** User guides are multi-page sphinx websites. I imagine that operational manuals will be presented the same way.
3. **Access.** User guides are written and published in the open. I suppose there could be some need to require authentication to see operations guides. If so we should identify that so we can plan to build a 'secured' variant of LSST the Docs.

Does this capture your concern? Or are you thinking of different differences?